SonarQube.Bootstrapper/ArgumentProcessor.cs

```diff
@@ -107,11 +115,11 @@ public static bool TryProcessArgs(string[] commandLineArgs, ILogger logger, out
 
                 if (phase == AnalysisPhase.PreProcessing)
                 {
-                    settings = CreatePreProcessorSettings(baseChildArgs, properties, globalFileProperties, logger);
+                    settings = CreatePreProcessorSettings(baseChildArgs, properties, globalFileProperties, installLoaderTargets, logger);
                 }
                 else
                 {
-                    settings = CreatePostProcessorSettings(baseChildArgs, properties, logger);
+                    settings = CreatePostProcessorSettings(baseChildArgs, properties, installLoaderTargets, logger);
```

Comment:
It doesn't make sense to support the argument in the post-process phase; we're silently ignoring it. The ideal solution would be to warn the user we're ignoring it, but it isn't worth changing now.